provision/core/git/provision.sh

```diff
@@ -58,5 +58,7 @@ function git_after_packages() {
     vvv_info " * Git hasn't been told how to merge branches, setting pull.rebase false for the merge strategy"
     noroot git config --global pull.rebase false
   fi
+  git config --global --add safe.directory '*' # Allow git to work well under docker provider
+  noroot git config --global --add safe.directory '*' # Allow git to work well under docker provider
```

Comment:
these might be needed regardless of wether we use docker or not, could these be a separate PR?